manifests/ops-files/windows/enable-rdp.yml

```diff
@@ -0,0 +1,26 @@
+- type: replace
+  path: /releases/-
+  value:
+    name: "windows-utilities"
+    version: "0.11.0"
+    url: "https://bosh.io/d/github.com/cloudfoundry-incubator/windows-utilities-release?v=0.11.0"
+    sha1: "efc10ac0f4acae23637ce2c6f864d20df2e3a781"
+
+- type: replace
+  path: /addons/-
+  value:
+    name: set-pw
+    include:
+      stemcell:
+      - os: windows1803
+    jobs:
+    - name: enable_rdp
+      release: windows-utilities
+      properties:
+        enable_rdp:
+          enabled: true
+    - name: set_password
+      release: windows-utilities
+      properties:
+        set_password:
+          password: ((windows-rdp-password))
```

Comment:
@benmoss Where is this password defined, and how should we go about setting it?

Reply:
It's an arbitrary variable left up to the user to set. You could add a variable to have BOSH generate it but we haven't been doing it that way.

Reply:
aka `bosh deploy -v windows-rdp-password=foobar`

Reply:
The ops-file is useful in development because it allows users to RDP into the bosh windows VMs. It does not need to be used by PKS. This variable is the password that would be used to log into the VM via RDP.